guides/common/modules/proc_installing-standalone-hammer.adoc

```diff
@@ -4,74 +4,124 @@
 
 .Prerequisites
-ifdef::katello,orcharhino,satellite[]
-* Ensure that you register the host to {ProjectServer} or {SmartProxyServer}.
-For more information, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
-* Ensure that the following repositories are enabled and synchronized on {ProjectServer}:
-ifdef::satellite[]
-** {RepoRHEL8BaseOS}
-** {RepoRHEL8AppStream}
-** {RepoRHEL8ServerSatelliteUtils}
+ifndef::katello,orcharhino,satellite[]
+* Ensure that the CA certificate of {ProjectServer} is deployed to the truststore on the host.
 endif::[]
-ifdef::katello,orcharhino[]
+ifdef::katello[]
+* Ensure that you register the host to {ProjectServer} or {SmartProxyServer}.
+* If you are installing on {EL}{nbsp}9, ensure that the following repositories are enabled and synchronized on {ProjectServer}:
+** {EL} 9 BaseOS
+** {EL} 9 AppStream
+** https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm[{Project} release RPM]
+* If you are installing on {EL}{nbsp}8, ensure that the following repositories are enabled and synchronized on {ProjectServer}:
 ** {EL} 8 BaseOS
 ** {EL} 8 AppStream
-endif::[]
-ifdef::katello[]
 ** https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm[{Project} release RPM]
 endif::[]
-
-+
-For more information, see {ContentManagementDocURL}Synchronizing_Repositories_content-management[Synchronizing Repositories] in _{ContentManagementDocTitle}_.
-endif::katello,orcharhino,satellite[]
-ifndef::katello,orcharhino,satellite[]
-* Ensure that the CA certificate of {ProjectServer} is deployed to the truststore on the host.
+ifdef::orcharhino[]
+* Ensure that you register the host to {ProjectServer} or {SmartProxyServer}.
+* If you are installing on {EL}{nbsp}9, ensure that the following repositories are enabled and synchronized on {ProjectServer}:
+** {EL} 9 BaseOS
+** {EL} 9 AppStream
+* If you are installing on {EL}{nbsp}8, ensure that the following repositories are enabled and synchronized on {ProjectServer}:
+** {EL} 8 BaseOS
+** {EL} 8 AppStream
+endif::[]
+ifdef::satellite[]
+* Ensure that you register the host to {ProjectServer} or {SmartProxyServer}.
+* If you are installing on {EL}{nbsp}9, ensure that the following repositories are enabled and synchronized on {ProjectServer}:
+** {RepoRHEL9BaseOS}
+** {RepoRHEL9AppStream}
+** {RepoRHEL9ServerSatelliteUtils}
+* If you are installing on {EL}{nbsp}8, ensure that the following repositories are enabled and synchronized on {ProjectServer}:
+** {RepoRHEL8BaseOS}
+** {RepoRHEL8AppStream}
+** {RepoRHEL8ServerSatelliteUtils}
 endif::[]
 
 .Procedure
+ifdef::foreman-deb[]
 . Enable the required repositories on the host.
-ifdef::katello,orcharhino,satellite[]
-For more information, see {ManagingHostsDocURL}Enabling_Custom_Repositories_on_Content_Hosts_managing-hosts[Enabling repositories on hosts] in _{ManagingHostsDocTitle}_.
++
+:distribution-codename: My_Distribution_Codename
```

Comment:
This could be a problem to overwrite the attribute. Could you add "parent-distribution-codename: {distribution-codename}" and then reset+unset it afterwards?

Reply:
How about I just drop the attribute to simplify things and use hard-coded `_My_Distribution_Codename_` in the command? I don't see any purpose in using the attribute actually, it was already there when I started updating the module. Or am I missing something @maximiliankolb?

Reply:
Yes, works for me: https://theforeman-foreman-documentation-preview-pr-3334.surge.sh/nightly/Hammer_CLI/index-foreman-deb.html#installing-standalone-hammer